scenario/deletesheet.go

```diff
@@ -20,7 +21,7 @@ type (
 	DeleteSheetSettings struct {
 		DeletionMode SheetDeletionModeEnum `json:"mode" displayname:"Deletion mode" doc-key:"deletesheet.mode"`
 		Title        string                `json:"title" displayname:"Sheet title" doc-key:"deletesheet.title"`
-		ID           string                `json:"id" displayname:"Sheet ID" doc-key:"deletesheet.id"`
+		ID           string                `json:"id" displayname:"Sheet ID" doc-key:"deletesheet.id" appstructure:"sheet"`
```

Comment:
same here, should probably be `appstructure:"object"`

Reply:
What, why? Deletesheet should only display objects of "sheet" type.

Reply:
wrote this before my comment on changesheet.